src/main.rs

```diff
@@ -21,42 +20,55 @@ mod process;
 mod setup;
 mod utils;
 
-fn setup_logger(is_daemon: bool, is_verbose: bool) {
-    let filter = if is_verbose {
-        LevelFilter::Trace
-    } else {
-        LevelFilter::Info
+enum LogTarget {
+    Terminal,
+    Syslog,
+}
+
+fn setup_logger(log_target: LogTarget, log_level: LevelFilter) {
+    let logger = fern::Dispatch::new().level(log_level);
+
+    let logger = match log_target {
+        LogTarget::Terminal => logger.chain(std::io::stdout()),
+        LogTarget::Syslog => {
+            let log_format = syslog::Formatter3164 {
+                facility: syslog::Facility::LOG_DAEMON,
+                hostname: None,
+                process: "spotifyd".to_owned(),
+                pid: 0,
+            };
+            logger.chain(syslog::unix(log_format).expect("Couldn't initialize logger"))
+        }
     };
 
-    if is_daemon {
-        syslog::init(syslog::Facility::LOG_DAEMON, filter, Some("Spotifyd"))
-            .expect("Couldn't initialize logger");
-    } else {
-        let logger_config = ConfigBuilder::new()
-            .set_level_padding(LevelPadding::Off)
-            .build();
-
-        TermLogger::init(filter, logger_config.clone(), TerminalMode::Mixed)
-            .map_err(Box::<dyn Error>::from)
-            .or_else(|_| SimpleLogger::init(filter, logger_config).map_err(Box::<dyn Error>::from))
-            .expect("Couldn't initialize logger");
-    }
+    logger.apply().expect("Couldn't initialize logger");
 }
 
 fn main() {
     let mut cli_config: CliConfig = CliConfig::from_args();
 
-    let is_daemon = !cli_config.no_daemon;
-    let is_verbose = cli_config.verbose;
-    setup_logger(is_daemon, is_verbose);
+    let is_deamon = !cli_config.no_daemon;
```

Comment:
It's actually called [daemon](https://en.wikipedia.org/wiki/Daemon_(computing)).